src/HttpJsonRpcRequestFactory.php

```diff
@@ -29,7 +29,7 @@ public function notification(string $method, ?array $params = null) : RequestInt
         return $this->createRequest(
             self::METHOD_NOTIFICATION,
             [
-                'jsonrpc' => self::SPEC,
+                'jsonrpc' => self::V2_0,
```

Comment:
tbh stejně ta libka umí jen v2.0, možná bych to vůbec nedělal jako konstantu, resp. max privátní ... nebo vidíš nějaký smysl v tom?